scripts/kibana-proxy.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
+
+"""
+Runs Kibana and aws-signing-proxy locally. The latter is used to sign requests by the former and forward them to an
+Amazon Elasticsearch instance. The default instance is the main ES instance for the current DSS stage.
+
+To install Kibana and aws-signing-proxy follow these steps (macOS only):
+
+1) Run
+
+   brew install golang
+
+2) As instructed by brew, set GOPATH and GOROOT, then set PATH to include $GOPATH/bin
+
+3) Run
+
+   go get github.com/cllunsford/aws-signing-proxy
+
+4) Install a matching version of Kibana (currently 5.5). Finding the homebrew-core commit for that version is the
+   trickiest part of the setup. It is possible that you don't need to do this as you might already have a matching
+   version of Kibana installed locally. I used the following command to install Kibana 5.5.2
+
+   brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/eb26ed9e7a4b3a35c62a40fa4fec89bf0361781f/Formula/kibana.rb
+
+To use this program, set `AWS_PROFILE` and `source environment`. Then run
+
+   kibana-proxy.py
+
```

Comment:
What is the appropriate way to cleanly terminate the Kibana proxy setup? Simply ctrl-C in the terminal window from which it was started, or something else?

Reply:
Ctrl-C does it. Will mention this.